recipes/default.rb

```diff
@@ -18,23 +18,37 @@
 #
 include_recipe 'chef-sugar'
 
-package 'apt-transport-https' if ubuntu?
+# setup docker apt repo
+include_recipe 'chef-apt-docker'
 
-apt_repository 'docker' do
-  uri 'https://apt.dockerproject.org/repo'
-  trusted true
-  components ["ubuntu-#{node['lsb']['codename']}", 'main']
-  only_if { ubuntu? }
+# IAM role support for tasks
+include_recipe 'sysctl::default'
+include_recipe 'iptables::default'
+sysctl_param 'net.ipv4.conf.all.route_localnet' do
+  value 1
+end
+iptables_rule 'dnat_80_to_51679' do
```

Comment:
Docker will add/remove iptables rules according to the containers requiring port forwarding. So, hardcoding the Docker rules isn't an option.
I'm not sure how can I have docker restart when iptables are reloaded via notification.

I ended up removing these iptables rules for now. We are not using IAM for Tasks right now. So, not a blocker. Docker not persisting it's rules is not nice